src/app/application.cpp

```diff
@@ -512,6 +512,7 @@ void Application::initializeTranslation()
     Preferences* const pref = Preferences::instance();
     // Load translation
     QString localeStr = pref->getLocale();
+    QLocale::setDefault(QLocale(localeStr));
```

Comment:
Don't we need to reset this if user changes translation during runtime?
Also will this affect the printing of thousand/decimal separators? I think the user would prefer his system defined separators.